src/component/Game.js

```diff
@@ -1,11 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Board from './Board';
 
 function Game() {
+  const [start, setStart] = useState(false);
   return (
     <div className="game">
       <div className="board">
-        <Board text = "this is text that you are typing in a typing test"/>
+        <Board 
+          text = "this is text that you are typing in a typing test"
+          setStart = {setStart}
+          start = {start}
+        />
```

Comment:
Could you help further rationalize why we are pulling this up a level?

It seems odd to define state that is never used locally.

Reply:
@jgayfer 
It feels like the `Board` component has too much control over the game and is itself doing too much. I would argue that `Board` should not track game state ie. Board should be in charge of rendering components *based* on game state.

By moving control of the game state to a higher level and out of Board I can better write tests for `Board`. You'll also see in the newest commit that there's an end-game boolean as well. I believe that I can further add more control to the `Game` component to make testing easier.

Reply:
My main concern is that our `Board` component needs several different states to function, but they're defined in two different components.

I'm fine with having state being passed down, and if anything would encourage it, but if this is the approach that we're wanting to take, I'd argue that all of our state should be defined at the `Game` level.

The underlying principle here is the notion of a presentational component and a functional component. If we moved to make `Game` a truly functional component, we could likely clean up the props that `Board` requires significantly, as well as its containing logic.

Reply:
Here's an example:

Functional (all state is defined here):
https://github.com/JuulLabs/juulio/blob/master/app/javascript/business/partners/components/warranty/PhoneVerificationModalPageController.js

Presentational (no state definitions, all logic is a presentational concern):
https://github.com/JuulLabs/juulio/blob/master/app/javascript/business/partners/components/warranty/PhoneVerificationModalPage.js